README.md

```diff
@@ -21,6 +21,8 @@ If a feature is not directly used to implement the node proxy component in ambie
 
 ## Building
 
+Please use Rust 1.70+.
```

Comment:
Can we have a single source of truth here? `Cargo.toml` has a `rust-version` field already. Probably its out of date, though

Reply:
My preference here (and what I do internally) is to say

"please use the Rust version that ships with the repo's `build-tools` image"

We can automate running build-tools and dump the `rustc` version to console with a make target if we wanna get fancy.

Reply:
I miss it, let me update doc, thanks.